pkg/apis/core/types.go

```diff
@@ -2571,6 +2571,12 @@ type PodSpec struct {
 	// More info: https://github.com/kubernetes/community/blob/master/keps/sig-network/0007-pod-ready%2B%2B.md
 	// +optional
 	ReadinessGates []PodReadinessGate
+	// RuntimeClassName refers to a RuntimeClass object with the same name, which should be used to
+	// run this pod.  If no RuntimeClass resource matches the named class, the pod will not be run.
```

Comment:
Document what happens when this is unset.

Reply:
Done